Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -340,6 +445,50 @@ namespace AzToolsFramework::Prefab
         return m_rootAliasFocusPathLength;
     }
 
+    PrefabEditScope PrefabFocusHandler::GetPrefabEditScope([[maybe_unused]] AzFramework::EntityContextId entityContextId) const
+    {
+        switch (m_openInstanceMode)
+        {
+        case 1:
+        {
+            return m_prefabEditScope;
+            break;
+        }
```

Comment:
@AMZN-daimini , this is probably not needed right? Based on previous discussions, we don't want to show the toggle to switch between 2 modes and make instance editing the default when overrides support is added. So, we should only have 2 states here?

Reply:
There's a couple reasons why we probably still want this in:
1) In the transition phase, we will need to be able to toggle between the two modes based on the setreg property that switches override UX on;
2) In the future, when we enable pinning, having an option to switch to the option where you can't expand un-focused prefabs could still be desirable, so we may as well just keep the piping there.

That said, we may want to rename the enums a bit to make the setting clearer.

Reply:
Yes, having the ability to take in the setreg value and showing one workflow or the other makes sense. The user would then need to restart the editor for the change to take effect. 

I think having the ability to switch between 2 modes in the editor might be a bit premature right now. Imo, it only makes sense if we have something to test with pinning. Otherwise, it's just added more code and logic to maintain till pinning arrives. If it is a small refactor, then I recommend removing the switching logic for now and adding it later when pinning development begins. 

If it's a large refactor or if you feel strongly about leaving this plumbing for ease of future testing, I'd be okay with getting this in its current state. It comes down to how much code we want to leave in for prototyping purposes.

Reply:
> That said, we may want to rename the enums a bit to make the setting clearer.

Discussed with @AMZN-daimini. Changed the enums to **SHOW/HIDE_NESTED_INSTANCES_CONTENT**.